### PR TITLE
Container-based sizing for GroupAvatar

### PR DESCRIPTION
**Pull request by bradencanderson**
This makes the `size` prop optional on Group Avatar.

I also updated the text sizing to be consistent with the single-collaborator Avatar case.

Before:
screen shot 2018-06-05 at 5 31 30 pm

After: 
screen shot 2018-06-05 at 5 31 37 pm

Avatar: 
screen shot 2018-06-05 at 5 32 21 pm

(The text sizing difference is more obvious if you open the docs from the different versions in 2 tabs and quickly toggle between them.)

### CODE REVIEW

**Comment by christianvuerings** [comment]
Deploy preview for *gestalt* ready!

Built with commit e7b1fadb226e10d7405cb6b02ab78c14eab44b2f

https://deploy-preview-244--gestalt.netlify.com

**Comment by pinterest-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Bundle size increase
  </td>
    </tr>
  </tbody>
</table>



Filename | Size Diff | Gzip Diff | Prev Size | Current Size | Prev Gzip | Current Gzip
--- | --- | --- | --- | --- | --- | --- |
gestalt.js | :small_red_triangle: 0.4% | :small_red_triangle: 0.8% | 266.77 KB | 267.96 KB | 58.92 KB | 59.4 KB
gestalt.es.js | :small_red_triangle: 0.5% | :small_red_triangle: 0.8% | 264.13 KB | 265.32 KB | 58.56 KB | 59.04 KB
  
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,11 +6,15 @@
...
 
+### Major
+* GroupAvatar: `size` prop is now optional. (#244)
+
 ### Minor
 * Masonry: Makes Masonry React Async compatible (#227)
 * SegmentedControl: Change flow type of `items` to `React.Node` (#230)
 * Video: Add jsdom browser specific tests (#205)
 * Card: Make Card explicitely use box-sizing: content-box (#243)
+* GroupAvatar: Text sizes have shrunk to be consistent with Avatar. (#244)
```
Comment (christianvuerings) [inline CHANGELOG.md]: @bradencanderson could you reach out to @jaymarsh from PDS to get a 👍 from design? We should also update PDS if we changed these sizes

File: docs/src/GroupAvatar.doc.js [hunk 2]
```diff
@@ -130,4 +130,30 @@ card(
   </Combination>
 );
 
+card(
+  <Example
+    description={`
+    GroupAvatars that are not given a \`size\` prop will expand to fit to the width of their
+    parent container. A common use case is to achieve column-based sizing.
+
+    Resize the browser to see these GroupAvatar change to match the width of the \`Column\` they
```
Comment (christianvuerings) [inline docs/src/GroupAvatar.doc.js]: @bradencanderson love this example - thanks for putting it together

**Pull request merged**